### PR TITLE
Create buddhism-trees-and-forests_dhammika.md

### PR DESCRIPTION
**Pull request by JJM317**
This was a toughie. I couldn't find any info or copy online, so I left out the external and source links, just putting in the drive link.

Also, it seems that the title is for a book and the an essay found in that book, so I put this as an essay.

Sorry that I couldn't do more with it. Let me know what I may have missed

### CODE REVIEW

File: _content/essays/buddhism-trees-and-forests_dhammika.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - dhammika
+drive_links:
+  - "https://drive.google.com/file/d/14Gp_ERzbNG0Ob8USGAUzO7pqx98uFuC1/view?usp=drive_link"
+formats: [pdf]
+course: nature
+tags:
+  - trees
+  - jataka
+booktitle: "Buddhism, Trees, and Forests"
+pages: "3"
```
Comment (buddhist-uni) [inline _content/essays/buddhism-trees-and-forests_dhammika.md:13]: yeah, it's really a 2 page essay, not including the cover. And page counts cannot be in quotes 😅

**Review by buddhist-uni — COMMENTED** [review]
Bhante Dhammika's BudBlooms site is linked from the sources page, but indeed it has terrible SEO so it's hard to Google. Anyway, here's the external url: https://budblooms.org/download/buddhism-trees-and-forests-pdf/

Unfortunately, Bhante doesn't specify the original publication info there, so usually I have to poke around his known haunts for the source url... In this case: https://www.sundaytimes.lk/160320/plus/respect-for-trees-not-worship-in-the-time-of-the-buddha-186819.html

Which gives us `year: 2016` and `month: mar` 😁

**Comment by JJM317** [comment]
So, should this be moved to articles since it is a newspaper article? Right
now I have it under essay.

~Joe

File: _content/essays/buddhism-trees-and-forests_dhammika.md [hunk 2]
```diff
@@ -2,15 +2,19 @@
...
 formats: [pdf]
+year: 2016
+month: mar
 course: nature
 tags:
   - trees
   - jataka
-booktitle: "Buddhism, Trees, and Forests"
-pages: "3"
+booktitle: "The Sunday Times"
```
Comment (buddhist-uni) [inline _content/essays/buddhism-trees-and-forests_dhammika.md:16]: no need for booktitle at all

**Review by buddhist-uni — COMMENTED** [review]
I think `essays` is okay for these? 

While originally published as articles, many of these BudBlooms essays were significantly rewritten between then and now, so the "canonical" version is the (current, standalone) essay, rather than the original article.

That said, you can also move this one to articles if you wish, as it's pretty close. In which case `journal: "Sunday Times"` and `address: "Kandy"` 😁

**Pull request merged**